.github/workflows/integration-tests.yml

```diff
@@ -12,11 +12,13 @@ jobs:
       steps:
          - id: setup-minikube
            name: Setup Minikube
-           uses: manusa/actions-setup-minikube@v2.4.2
+           uses: medyagh/setup-minikube@latest
```

Comment:
Can we pin this to a sha rather than latest? That's pretty important for security.

Reply:
this appears to be resolve in the latest commit